packages/create-single-spa/bin/create-single-spa.js

```diff
@@ -1,8 +1,36 @@
 #!/usr/bin/env node
 
+checkNodeVersion();
+
 const yeoman = require("yeoman-environment");
 const argv = require("yargs").argv;
 
 const env = yeoman.createEnv();
 env.registerStub(require("generator-single-spa"), "single-spa");
-env.run(`single-spa ${argv._.join(" ")}`, argv);
```

Comment:
who knows - maybe someone will be using a version of node that doesn't support template strings. Poor souls